### PR TITLE
add a snitch for retire instances job

### PR DESCRIPTION
**Pull request by hchen15**
switch from using email to snitches to alert for job failure

### CODE REVIEW

File: devops/jobs/MinosLifecycle.groovy [hunk 1]
```diff
@@ -87,6 +88,7 @@ class MinosLifecycle {
                         env('ENVIRONMENT', environment)
                         env('DEPLOYMENT', deployment)
                         env('AWS_REGION',extraVars.get('AWS_REGION','us-east-1'))
+                        env('SNITCH', inner_config.get('snitch', ''))
```
Comment (feanil) [inline devops/jobs/MinosLifecycle.groovy]: If this is empty, curl will fail in the bash script.  It seems like we should add an assert to ensure the snitch url is set?

Comment (jibsheet) [inline devops/jobs/MinosLifecycle.groovy]: I've normally done if (snitch) { shell('curl $snitch') } which achieves similar goals of not blowing up

Comment (jibsheet) [inline devops/jobs/MinosLifecycle.groovy]: (rabbit for example can run without the snitch)

Comment (feanil) [inline devops/jobs/MinosLifecycle.groovy]: I'm fine with making this an optional setting but then the bash script should be updated.

**Pull request merged**